modules/reliability/templates/menu_reliability.tpl

```diff
@@ -1,96 +1,93 @@
+                      <!-- {* <td>Objective:</td>
+                          <td>{$form.Objective.html}</td> *} -->
+                          <label class="col-sm-12 col-md-4">{$form.CenterID.label}</label>
+                          <div class="col-sm-12 col-md-8">{$form.CenterID.html}</div>
+                      </div>
+                      <div class="form-group col-sm-6">
+                          <label class="col-sm-12 col-md-4">{$form.PSCID.label}</label>
+                          <div class="col-sm-12 col-md-8">{$form.PSCID.html}</div>
+                      </div>
+                  </div>
+                  <div class="row">
+                      <div class="form-group col-sm-6">
+                          <label class="col-sm-12 col-md-4">{$form.Instrument.label}</label>
+                          <div class="col-sm-12 col-md-8">{$form.Instrument.html}</div>
+                      </div>
+                      <div class="form-group col-sm-6">
+                          <label class="col-sm-12 col-md-4">{$form.reliability_center_id.label}</label>
+                          <div class="col-sm-12 col-md-8">{$form.reliability_center_id.html}</div>
+                      </div>
+                  </div>
+                  <div class="row">
+                      <div class="form-group col-sm-6">
+                          <label class="col-sm-12 col-md-4">{$form.Gender.label}</label>
+                          <div class="col-sm-12 col-md-8">{$form.Gender.html}</div>
+                      </div>
+                      <div class="form-group col-sm-6">
+                          <label class="col-sm-12 col-md-4">{$form.Visit_label.label}</label>
+                          <div class="col-sm-12 col-md-8">{$form.Visit_label.html}</div>
+                      </div>
+                  </div>
+                  <div class="row">
+                      <div class="form-group col-sm-6">
+                          <label class="col-sm-12 col-md-4 label-control">{$form.Invalid.label}</label>
+                          <div class="col-sm-12 col-md-8">{$form.Invalid.html}</div>
+                      </div>
+                      <div class="form-group col-sm-6">
+                          <label class="col-sm-12 col-md-4">{$form.ProjectID.label}</label>
+                          <div class="col-sm-12 col-md-8">{$form.ProjectID.html}</div>
+                      </div>
+                  </div>
 
+                  <div class="row">
+                      <div class="col-sm-4 col-md-3 col-xs-12 col-md-offset-5 col-sm-offset-4">
+                          <input type="submit" name="filter" value="Show Data" class="btn btn-sm btn-primary col-xs-12" />
+                      </div>
 
+                      <div class="visible-xs col-xs-12"> </div>
+                      <div class="visible-xs col-xs-12"> </div>
+                      <div class="visible-xs col-xs-12"> </div>
+                      <div class="visible-xs col-xs-12"> </div>
```

Comment:
It would, but this particular update only addresses the bug described in the redmine ticket.
Implementing react data table was just the quickest solution for that.